api/app/signals/apps/questionnaires/models/question_graph.py

```diff
@@ -30,6 +29,22 @@ def _set_model_order(self, question, model, ids):
 
         return model.objects.filter(graph=self, question=question).values_list('id', flat=True)
 
+    def get_nodes(self):
```

Comment:
As discussed on Jitsi, this function is not needed. We can write all metadata to nodes in the loop in `api/app/signals/apps/questionnaires/services/question_graph.py::QuestionGraphService::_build_nx_graph`.

I provided a patch with the required change.